src/components/WwwFrame/TheTipMessage.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+	<transition name="fade">
+		<div v-if="tipVisible" class="message-text text-center small-12" :class="typeClass">
+			<span class="message-content">
+				<div class="icon-wrapper">
+					<kv-icon :name="iconName" />
+				</div>
+				<p class="message">{{ tipMsg }}</p>
+			</span>
+			<a @click="closeTip" class="close-tip-message" aria-label="Close">
+				<kv-icon name="x" />
+			</a>
+		</div>
+	</transition>
+</template>
+
+<script>
+import _get from 'lodash/get';
+import tipMessageData from '@/graphql/query/tipMessageData.graphql';
+import updateTipMessage from '@/graphql/mutation/updateTipMessage.graphql';
+import KvIcon from '@/components/Kv/KvIcon';
+
+export default {
+	components: {
+		KvIcon
+	},
+	inject: ['apollo'],
+	data() {
+		return {
+			tipVisible: false,
+			tipPersist: false,
+			tipInitUrl: '',
+			tipMsg: '',
+			tipMsgType: ''
+		};
+	},
+	computed: {
+		typeClass() {
+			// Valid options are 'warning' + 'error' otherwise use confirmation as default
+			if (this.tipMsgType === 'warning' || this.tipMsgType === 'error') {
+				return `message-text-${this.tipMsgType}`;
+			}
+			return 'message-text-confirmation';
+		},
+		iconName() {
+			// warning icon-info, error icon-error
+			if (this.tipMsgType === 'warning') {
+				return 'info';
+			} else if (this.tipMsgType === 'error') {
+				return 'error';
+			}
+			// default icon-confirmation
+			return 'confirmation';
+		}
+	},
+	methods: {
+		/*
+			Close tip message
+			- tipMsg + tipMsgType are retained when closed by clicking 'X'
+		*/
+		closeTip() {
+			this.apollo.mutate({
+				mutation: updateTipMessage,
+				variables: {
+					tipMsg: this.tipMsg,
+					tipMsgType: this.tipMsgType,
+					tipVisible: false,
+					tipPersist: false,
+					tipInitUrl: ''
+				}
```

Comment:
I would think that the only variable you would want to set for this mutation is `tipVisible: false`. Maybe the resolver should be changed so that it merges in variables instead of just replacing them from the query. Or maybe you could use some default values in the graphql query?